src/patternfly/components/Chip/examples/chip-example.hbs

```diff
@@ -2,7 +2,8 @@
   {{#> chip-text chip-text--attribute='id="chip_one"'}}
     Chip
   {{/chip-text}}
-  {{#> button button--modifier="pf-m-plain" button--HasTimesCircleIcon="true" button--attribute='aria-labelledby="remove_chip_one chip_one" aria-label="Remove" id="remove_chip_one"'}}
+  {{#> button button--modifier="pf-m-plain" button--attribute='aria-labelledby="remove_chip_one chip_one" aria-label="Remove" id="remove_chip_one"'}}
+    {{> button-icon button-icon--type="times"}}
```

Comment:
This is the wrong icon. Compare to pf-next to see what the icon should look like - https://pf-next.com/components/Chip/examples/

Reply:
Correct button-icon for chip/chipgroup has been applied.